contracts/contracts/ProviderRegistry.sol

```diff
@@ -159,21 +159,19 @@ contract ProviderRegistry is
         uint256 residualBidPercentAfterDecay
     ) external nonReentrant onlyPreConfirmationEngine {
         uint256 residualAmt = (amt * residualBidPercentAfterDecay * PRECISION) / PERCENT;
-        require(providerStakes[provider] >= residualAmt, "Insufficient funds to slash");
-        providerStakes[provider] -= residualAmt;
+        uint256 penaltyFee = (residualAmt * uint256(feePercent) * PRECISION) / PERCENT;
```

Comment:
We should call this var protocolFee or treasuryFee

Reply:
So the term "protocol fee" is not applicable to the provider registry anymore? And (to confirm) "protocol fee" is still relevant to the bidder registry?

If this is the case, we should replace all instances of "protocol fee" with "penalty fee" in this file